install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -9,9 +9,9 @@ Create chart name and version as used by the chart label.
 Return the appropriate apiVersion for ingress.
 */}}
 {{- define "ingress.apiVersion" -}}
-{{- if semverCompare ">=1.4-0, <1.14-0" .Capabilities.KubeVersion.GitVersion -}}
+{{- if semverCompare ">=1.4-0, <1.19-0" .Capabilities.KubeVersion.Version -}}
```

Comment:
> NetworkPolicy in the extensions/v1beta1 API version is no longer served
> Migrate to use the networking.k8s.io/v1 API version, available since v1.8. Existing persisted data can be retrieved/updated via the new version.

Based on this [docs](https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/), extensions/v1beta1 will not be available for kube >1.16, so I think the change for 1.19 will cause some issue.

Note: The original check with 1.14 is not so strict, but i think it will work as well :)